filewriter_v2/README.md

```diff
@@ -0,0 +1,388 @@
+
+An _NXdetector_channel_ group could contain every field that is allowed for NXdetector and that makes sense to be given per channel.
+
+[^2]: Currently, _AXISNAME_ is defined to be of type [_NX_NUMBER_](https://manual.nexusformat.org/nxdl-types.html#nx-number), but there is an [open PR](https://github.com/nexusformat/NIAC/issues/97) to allow [_NX_CHAR_](https://manual.nexusformat.org/nxdl-types.html#nx-char) as well.
+
+#### Example
+
+For example, if `threshold_1`, `threshold_2` and `difference_mode` are enabled, we would get the following:
+
+```
+data: NXdata
+  @signal = "data"
+  @axes = ["image_number", "channel", ".", "."]
+  @image_number_indices = 0
+  @channel_indices = 1
+  image_number = [1, ..., number_of_images]
+  channel = ["threshold_1", "threshold_2", "difference"]
+  data = uint[height, width]
+
+detector: NXdetector
+  ...
+  threshold_1_channel: NXdetector_channel
+    threshold_energy = float
+    flatfield = float[height, width]
+    pixel_mask = uint[height, width]
+  threshold_2_channel: NXdetector_channel
+    threshold_energy = float
+    flatfield = float[height, width]
+    pixel_mask = uint[height, width]
+  difference_channel: NXdetector_channel
+    threshold_energy = float[2]
+```
+
+## List of fields
+
+A file generated with in V2 format may contain the fields and attributes listed below. For further details and descriptions please refer to the provided links to the NeXus definitions.
+
+### / (root)
+
+| Name | NeXus Type | Description |
+|---|---|---|
+| [@default](https://manual.nexusformat.org/classes/base_classes/NXroot.html?highlight=nxroot#nxroot-default-attribute) | NX_CHAR | Declares which NXentry group contains the data to be shown by default. |
+
+### /entry
+
+The top-level NeXus group which contains all the data and associated information that comprise a single measurement.
+
+| Name | NeXus Type | Description |
+|---|---|---|
+| @NX_class | NX_CHAR | [`"NXentry"`](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-group)
+| [@default](https://manual.nexusformat.org/classes/base_classes/NXentry.html#nxentry-default-attribute) | NX_CHAR | Declares which NXdata group contains the data to be shown by default. |
+| [@version](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-version-attribute) | NX_CHAR | Describes the version of the NXmx definition used to write this data. |
+| [definition](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-definition-field) | NX_CHAR | NeXus NXDL schema to which this file conforms, `"NXmx"`. |
+| [end_time_estimated](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-end-time-estimated-field) | NX_DATE_TIME | Estimated time/date of the last data point collected in UTC, following ISO 8601 with suffix Z. Note that the time zone of the beamline is provided in `/entry/instrument/time_zone`. |
```

Comment:
```suggestion
| [end_time_estimated](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-end-time-estimated-field) | NX_DATE_TIME | Estimated time/date of the last data point collected in UTC, following ISO 8601 with suffix Z. Note that the time zone of the beam-line is provided in `/entry/instrument/time_zone`. |
```

Reply:
"beamline" is correct. See: https://en.wikipedia.org/wiki/Beamline

Reply:
Some here, won't change since it is correct